scene/gui/button.cpp

```diff
@@ -109,6 +109,13 @@ Ref<StyleBox> Button::_get_current_stylebox() const {
 
 void Button::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE: {
+			// Update text with the current translation state.
+			if (!text.is_empty() && get_auto_translate_mode() != AUTO_TRANSLATE_MODE_DISABLED) {
+				notification(NOTIFICATION_TRANSLATION_CHANGED);
```

Comment:
This looks like something that should be sent automatically, like `NOTIFICATION_THEME_CHANGED`.

Reply:
Wouldn't that mean that every UI node with text would redraw on tree entering, even if it didn't need to?

Reply:
Well, every CanvasItem already redraws on tree entering, so it doesn't make difference.

Reply:
Alright then, gonna need to know why this method in your old PR is here though: https://github.com/godotengine/godot/pull/75012/files#diff-2b7ba034670859ced66e442b9ec59f7351febc3b4352e535bc85da804bcca266R2918

Reply:
The comment explains it 🙃
Nodes on the scene shouldn't be translated, because it would make editing them more difficult.

Reply:
...I meant the combination of `set_message_translation(false)` and `notification(NOTIFICATION_TRANSLATION_CHANGED)`. Doesn't those two cancel each other out?

Reply:
I think the TRANSLATION_CHANGED was to force the texts to update, as some of them can be set before entering tree. I don't remember if it actually fixed something, but I'd assume it did.

If the translations are now part of Node, this code should be moved.